Vienna/Sources/Main window/FolderTreeCellView.h

```diff
@@ -0,0 +1,32 @@
+//
+//  FolderTreeCellView.h
+//  Vienna
+//
+//  Copyright 2021 Joshua Pore
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  https://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import <Cocoa/Cocoa.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FolderTreeCellView : NSTableCellView
+
+@property (nonatomic) BOOL inProgress;
+@property (nonatomic) NSInteger unreadCount;
+@property (nonatomic) BOOL didError;
```

Comment:
Nitpicking because you asked for comments, feel free to ignore: This variable name seems odd. The tree cell view probably did not make an error, so who did it? Maybe name it "showError" "displayError" are better choices for that variable name.

Reply:
Thanks! You're correct, `showError` probably makes more sense, but I wonder if that implies that it would be possible to have an error but not show it?
The associated image view should probably be renamed from `auxiliaryImageView` as well.